ManatiCR/vitaminau
# VIT-82: Changes work in progress form.

# Task [VIT-82](https://manati.atlassian.net/browse/VIT-82).
Changes in work in progress form.

# Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the following URL: 
  - http://vitaminau.docker/formularios/trabajo-en-curso
 Note: Lean on the ticket: 
- [VIT-82](https://manati.atlassian.net/browse/VIT-82).
  - Verify that the introductory text of the "Actividad" fieldset is: Agregue actividades en las que existen beneficiarios externos, más adelante se le solicitará la información de los beneficiarios por actividad.
  - In the previous field called "Número de actividad", the label should be: "Coloque la cantidad de actividades que ingresó anteriormente" and the description should be "Número de actividad"
- [VIT-78](https://manati.atlassian.net/browse/VIT-78).
  - In the "Beneficiarios" section, in the "Persona individual" field, when selecting "Con especificaciones" you must display a text area "Especificaciones".
  - In the section of "Beneficiarios" when selecting "Comunidad" the text area "Detalle comunidad" should have the help text "Escriba el nombre de los distritos u otras señas."
  - In the "Beneficiarios" section, when selecting "Comunidad" and selecting any province, you must display a help text at the top where the cantons are displayed that says "Para seleccionar múltiples cantones mantenga presionada la tecla ctrl (si está en Linux o Windows) o la tecla command (si está en Mac OS)” y de click en los cantones que necesita."

# CA
- Changes must be shown in the 15 conditional beneficiary fields.
- Check the improvement in the introductory text of the Activity fieldset.
- In the Activity number field, turn the label: which label becomes the name of the field and the name of the field becomes the label.
- Check beneficiaries:
  - Markup help text in cantons and help text in community detail.
  - Options in individual person change -> Specific person to Specifications and the text area change to Specifications.
